ansible-ipi-install/roles/node-prep/tasks/10_power_off_cluster_servers.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Install required packages for ipmi_power module
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ ipmi_power_packages }}"
+
+- name: IPI-INSTALL Power OFF All Nodes
```

Comment:
this this is also a "per-cluster" action, it probably belongs in the installer steps as well.

Reply:
If we talking multiple clusters, yeah I would agree with that.